### ISSUE TITLE
Secondary block selection for conflict and hazard prevention.

### ISSUE THREAD

**Issue by crubleigh** [rebane2001/mapartcraft#82]
One issue I've run into time and time again is block conflicts. For me, dirt and grass are the cheapest materials for those colors however sometimes on the map they end up close enough together to spread the grass. I then have to scour the map for where the dirt and grass get close to each other and put down a sufficient number of jungle slabs. Another block conflict I've run into has been water adjacent to carpet, as this really doesn't work great in survival. Another thing to check for would be TNT next to pressure plates as that could become dangerous. My suggestion is that the program could detect these block conflicts and then ask for a secondary block in that color that would resolve the conflict. This way it could be baked into the schematic and I can still use mostly dirt but not have to go and look out for where grass might spread.

**Comment by rebane2001** [comment]
Water is unsupported and TNT can (and should!!) easily be replaced with redstone blocks.
I will be looking into grass however.

**Comment by crubleigh** [comment]
I see water on the site and it's come up before in a map, it worked fine when pasted in but obviously when I built it in survival I had to replace  carpets. Do you mean like the different shades of water based on depth?

**Comment by rebane2001** [comment]
Yes, if you hover over the "water" selection you can see it says unsupported
